### PR TITLE
Support CNAME entries

### PR DESCRIPTION
**Pull request by samgiles**
A CNAME entry is useful for "remote" (tunnel) name.

### CODE REVIEW

**Comment by samgiles** [comment]
@michielbdejong r?

**Comment by ferjm** [comment]
Should we add a test here https://github.com/fxbox/dns-server/blob/support-cname/test/dns-query_test.js#L37 ?

**Comment by samgiles** [comment]
These aren't run at the moment, but I will do.  Thanks @ferjm :)

File: src/server.js [hunk 1]
```diff
@@ -6,13 +6,15 @@ const dnsServer = dns.createServer();
...
+  5: 'CNAME',
   16: 'TXT'
 };
 const API_BASE = ['', 'v1', 'dns'];
 const RECORD_TTL = 600;
 let records = {
   TXT: {},
-  A: {}
+  A: {},
+  CNAME: {},
```
Comment (ferjm) [inline src/server.js:17]: IIUC with this change we store CNAME values, but we don't give them back on client queries, right? Should we add them to https://github.com/fxbox/dns-server/pull/8/files#diff-e6a5b42b2f7a26c840607370aed5301aR31

Comment (samgiles) [inline src/server.js:17]: We just return based on the request type: https://github.com/fxbox/dns-server/blob/support-cname/src/server.js#L26  Without this change, we get undefined errors when assigning to a type not seen before: https://github.com/fxbox/dns-server/blob/master/src/server.js#L98

Comment (ferjm) [inline src/server.js:17]: Ah, ok, makes sense. Thanks

**Pull request merged**